src/defs.h

```diff
@@ -10,64 +10,142 @@
 //upper inclination for geosynchronous satellites
 #define GEOSYNCHRONOUS_INCLINATION_THRESHOLD_DEGREES 70
 
-/* Constants used by SGP4/SDP4 code */
-#define	km2mi		0.621371		/* km to miles */
-#define deg2rad		1.745329251994330E-2	/* Degrees to radians */
-#define pi		3.14159265358979323846	/* Pi */
-#define pio2		1.57079632679489656	/* Pi/2 */
-#define twopi          6.28318530717958623     /* 2*Pi  */
-#define x3pio2		4.71238898038468967	/* 3*Pi/2 */
-#define e6a		1.0E-6
-#define tothrd		6.6666666666666666E-1	/* 2/3 */
-#define xj2		1.0826158E-3		/* J2 Harmonic (WGS '72) */
-#define xj3		-2.53881E-6		/* J3 Harmonic (WGS '72) */   
-#define xj4		-1.65597E-6		/* J4 Harmonic (WGS '72) */
-#define xke		7.43669161E-2
-#define xkmper		6.378137E3		/* WGS 84 Earth radius km */
-#define xmnpda		1.44E3			/* Minutes per day */
-#define ae		1.0
-#define ck2		5.413079E-4
-#define ck4		6.209887E-7
-#define f		3.35281066474748E-3	/* Flattening factor */
-#define ge		3.986008E5 	/* Earth gravitational constant (WGS '72) */
-#define s		1.012229
-#define qoms2t		1.880279E-09
-#define secday		8.6400E4	/* Seconds per day */
-#define omega_E		1.00273790934	/* Earth rotations/siderial day */
-#define omega_ER	6.3003879	/* Earth rotations, rads/siderial day */
-#define zns		1.19459E-5
-#define c1ss		2.9864797E-6
-#define zes		1.675E-2
-#define znl		1.5835218E-4
-#define c1l		4.7968065E-7
-#define zel		5.490E-2
-#define zcosis		9.1744867E-1
-#define zsinis		3.9785416E-1
-#define zsings		-9.8088458E-1
-#define zcosgs		1.945905E-1
-#define zcoshs		1
-#define zsinhs		0
-#define q22		1.7891679E-6
-#define q31		2.1460748E-6
-#define q33		2.2123015E-7
-#define g22		5.7686396
-#define g32		9.5240898E-1
-#define g44		1.8014998
-#define g52		1.0508330
-#define g54		4.4108898
-#define root22		1.7891679E-6
-#define root32		3.7393792E-7
-#define root44		7.3636953E-9
-#define root52		1.1428639E-7
-#define root54		2.1765803E-9
-#define thdt           4.3752691E-3
-#define rho            1.5696615E-1
-#define mfactor                7.292115E-5
-#define sr             6.96000E5       /* Solar radius - km (IAU 76) */
-#define AU             1.49597870691E8 /* Astronomical unit - km (IAU 76) */
+#include <float.h>
 
-/* doppler calculation */
-#define SPEED_OF_LIGHT 299792458.0
+/**
```

Comment:
I suggest we stick to one Doxygen comment style. I also suggest the triple slash style.

See https://www.stack.nl/~dimitri/doxygen/manual/grouping.html#memgroup for how to group these constants and document the groups.

Reply:
Fixed in 969fde5ee749ae7e09bd7cfc157bb441836c23b9